crypto/provider_core.c

```diff
@@ -134,8 +134,7 @@ static const OPENSSL_CTX_METHOD provider_store_method = {
 static CRYPTO_ONCE provider_store_init_flag = CRYPTO_ONCE_STATIC_INIT;
 DEFINE_RUN_ONCE_STATIC(do_provider_store_init)
 {
-    return OPENSSL_init_crypto(0, NULL)
-        && (provider_store_index =
```

Comment:
Why is this not needed?

Reply:
Lockup when the configuration file contains provider data.  Please, not needed just to create a lock.

Frankly, it was added "just 'cause", i.e. there was no real thought about from my end, so...